app/experimenter/kinto/client.py

```diff
@@ -31,6 +32,46 @@ def has_pending_review():
     return collection["data"]["status"] == KINTO_REVIEW_STATUS
 
 
+def get_rejected_collection_data():
+    client = kinto_http.Client(
+        server_url=settings.KINTO_HOST, auth=(settings.KINTO_USER, settings.KINTO_PASS),
+    )
+    collection = client.get_collection(
+        id=settings.KINTO_COLLECTION, bucket=settings.KINTO_BUCKET
+    )
+
+    if collection["data"]["status"] == KINTO_REJECTED_STATUS:
+        return collection["data"]
+
+
+def get_rejected_record():
+    client = kinto_http.Client(
+        server_url=settings.KINTO_HOST, auth=(settings.KINTO_USER, settings.KINTO_PASS),
+    )
+
+    main_records = client.get_records(
+        bucket=settings.KINTO_BUCKET_MAIN, collection=settings.KINTO_COLLECTION
+    )
+
+    workspace_records = client.get_records(
+        bucket=settings.KINTO_BUCKET, collection=settings.KINTO_COLLECTION
```

Comment:
maybe we should rename this to `KINTO_BUCKET_WORKSPACE` or something so it's clear waht the difference between this and `_MAIN` is